Makefile

```diff
@@ -204,6 +204,10 @@ ifneq ($(MM_NO_ENTERPRISE_LINT),true)
 endif
 endif
 
+app-layers: ## Extract interface from App struct
+	env GO111MODULE=on $(GO) get -u github.com/reflog/struct2interface
```

Comment:
Setiing `GO111MODULE=on` makes changes in `go.mod` when you run the target. I think it would be better to explicitly set this to off to mimic what we are doing with tools like `mmgotool` and prevent this.